src/openssl.c

```diff
@@ -4062,10 +4070,31 @@ static BIO *getbio(lua_State *L) {
 } /* getbio() */
 
 
+static int pem_pw_cb(char *buf, int size, int rwflag, void *u) {
```

Comment:
Maybe worth adding a comment of "expects nil/function/string on top of the stack" and "leaves one item on the top of the stack".

"errors from the function are not reported"

Reply:
perhaps it should leave nothing on the stack?

Reply:
Requested comments added.